source/administration/migrating.rst

```diff
@@ -26,4 +26,70 @@ The following instructions migrate Mattermost from one server to another by back
+
+This feature can be accessed through the `Mattermost Web App </administration/migrating.html#migrating-from-slack-using-the-mattermost-web-app>`_ or using the `CLI </administration/migrating.html#migrating-from-slack-using-the-mattermost-cli>`_.
+
+.. warning:: **It is highly recommended that you test Slack import before applying it to an instance intended for production.**
+
+   If you use Docker, you can spin up a test instance in one line (``docker run --name mattermost-dev -d --publish 8065:80 mattermost/platform``). If you don't use Docker, there are `step-by-step instructions <http://docs.mattermost.com/install/docker-local-machine.html>`_ to install Mattermost in preview mode in less than 5 minutes.
+
+Supported Features
+++++++++++++++++++
+
+The following key features are supported out of the box when importing from Slack:
+
+* User accounts with an email address set.
+
+* Public channels and the text messages posted in them by users.
+
+* Channel topic and purpose.
+
+* Automatically adding imported users to their channels.
+
+Messages with file attachments are imported as a message containing a link to Slack's servers by default. The file attachments themselves can be imported to Mattermost by using the `Slack Advanced Exporter <https://github.com/grundleborg/slack-advanced-exporter>`_ tool to add them to your archive before importing it.
+
+Bot and Integration messages are imported by default, but if you would like them to display with the appropriate username when imported, you should ensure that `Enable Integrations to Override Usernames </administration/config-settings.html#enable-integrations-to-override-usernames>`_ is set **before** doing the import.
+
+.. note:: Slack user accounts with the same email address as existing accounts on your Mattermost server will be merged into those accounts on import.
+
+Limitations
++++++++++++
+
+The following limitations are present when importing from Slack:
+
+* The Markdown subset used by Slack's Posts 2.0 feature announced on September 28, 2015 is not yet supported.
+
+* Direct Message and Private Groups cannot be imported. Slack does not include these messages when generating the export archive.
+
+Migrating from Slack using the Mattermost Web App
++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. note:: For larger imports, particularly those where you have used the `slack-advanced-exporter tool` to add Slack post attachments to the archive, it is recommended to import the Slack data using the `CLI </administration/migrating.html#migrating-from-slack-using-the-mattermost-cli>`_.
+
+1. Generate a Slack "Export" file from **Slack > Team Settings > Import/Export Data > Export > Start Export**.
+
+2. In Mattermost go to **Team Settings > Import > Import from Slack**. *Team Administrator* or *System Administrator* role is required to access this menu option.
```

Comment:
I don't think we usually italicize Team Admin and System Admin.

We also use "Team Admin" and "System Admin" when defining the roles in our docs: https://docs.mattermost.com/help/getting-started/managing-members.html#user-roles